### ISSUE TITLE
I believe the remote location of the Icelandic list was removed by mistake 

### ISSUE THREAD

**Issue by samolesen** [uBlock-LLC/uBlock#326]
In 69b6027174bcc788068a6f1a4ef3c2d936290e3e the remote location of the Icelandic list was removed.
In release 0.7.0.7 it is mentioned that is was because a user (me) reported it was whitelisting google ads. I was only mentioning the Danish list (which correctly had its remote location removed already), so the removal of the Icelandic list might have been a mistake, unless it was removed for some other reason.

There have apparently also been some progress with the Danish list now. The maintainer have today removed the whitelist of google ads from the Danish list. Whether it can trusted in the future as a remote location is another issue..

**Comment by gorhill** [comment]
You're right, I got really confused today it seems. I see I had removed the entry already with 3248b8d27c0308ef54de8a90e74f35c5d1d507fd (and properly without leaving a trailing comma), I forgot I had done so.

I will put it back.

**Comment by gorhill** [comment]
Wait a minute, I was right to remove the entry, I just got confused as to why I did when I scrambled to explain what happened while in panic mode.

There is a bug in the Icelandic list, which I fixed locally and I don't want people to suffer the bug, hence I have to remove the URL address of the remote version. I notified the author about the bug a while ago but it's not getting fixed.

The bug is: `gaflari.is#div(class*=auglysingaplass)`
My fix: `gaflari.is##div[class*="auglysingaplass"]`

The bug is that it is not a valid filter syntax and neither a valid CSS selector, I expect this will cause the content script to emit an exception and prevent uBlock from functioning properly.
